apps/api-documenter/package.json

```diff
@@ -31,7 +31,7 @@
     "@rushstack/heft-node-rig": "workspace:*",
     "@types/heft-jest": "1.0.1",
     "@types/js-yaml": "3.12.1",
-    "@types/node": "10.17.13",
+    "@types/node": "^12.20.24",
```

Comment:
Can you change these to just the version without the carat? Typings packages aren't as stable as other packages.

Reply:
Done! 👍